### DISCUSSION TITLE
Using Solito with Prisma and tRPC

### DISCUSSION THREAD

**Discussion by valerius21** [nandorojo/solito#148] [Q&A] [answered]
Hello there,

I am fiddling around with the Solito + TailwindCSS starter monorepo. After watching Fernandos talk on YouTube I was cruious about writing my app using the mentioned maximes. 

Since I had lots of success in previous projects with using tRPC in the past, I wanted to use it in my next project as well. 

After having a look at https://github.com/trpc/zart I was wondering if I could use tRPC with a Solito starter.

However, it became clear to me that I am unsure on how I am going to integrate prisma with tRPC into my new Solito project. 
So far, setting up prisma in the `packages/app` directory with `yarn add -D prisma && npx prisma init` resulted in some erros, e.g. it messes with the `solito` packge by being unable to resolve the dependency in the screens.

https://user-images.githubusercontent.com/10323945/189485712-720eaf48-3095-4c99-9e62-6980d6a51e77.png

Therefore, I was wondering on which is the best way on using those techs in a new solido project.

**Comment by nandorojo** [comment, marked as answer]
can you try running `yarn` from the root of the repo?

**Reply by valerius21** [reply]
It results in another error: 
![grafik](https://user-images.githubusercontent.com/10323945/189486891-4a7e3d1a-4539-4e2a-be8c-a3137d06d74e.png)

I've also retried it with a clean repo.

**Reply by nandorojo** [reply]
that’s pretty weird, i don’t know what prisma is doing to cause that

**Reply by nandorojo** [reply]
I think this error is related to #150 

Can you downgrade solito to 0.0.29?

**Reply by nandorojo** [reply]
Actually, `yarn add solito` with `1.0.11` should fix this.

**Reply by valerius21** [reply]
Hi @nandorojo ! Thank you for your fix! It resolved a lot of issues. I got solito with Tailwind, tRPC and Prisma working now.

Also many thanks for all the work you provided. It really helped me to get started with my project. Cheers!

**Reply by galacoder** [reply]
Hi @valerius21. I want to add this stack to my next project, but I don't know how to make it work yet. Could you please share a starter or some instruction how you did it. Your help is pretty much appreciated.

**Reply by valerius21** [reply]
> Hi @valerius21. I want to add this stack to my next project, but I don't know how to make it work yet. Could you please share a starter or some instruction how you did it. Your help is pretty much appreciated.

Create and add a new trpc context to the wrapper provider with a new prisma package. It takes a lot of more steps and distilling it down into a starter, but you should be fine by following the techs corresponding docs :)
